tests/routes/trips.js

```diff
@@ -61,4 +61,30 @@ describe('Trips API', function(){
       });
     });
   });
+  
+  describe('GET - /Trips', function(){
+    it('should return all trips in system', function(done){
+      request(app).get('/trips')
+      .expect(200)
+      .end(function(err, res) {
+        
+        assert.equal(res.body.data.length > 1, true);
```

Comment:
due to the mock test data, there will always be 1 trip. best to change this to:
`res.body.data.length > 0`